build/Dockerfile.scaphandre

```diff
@@ -0,0 +1,8 @@
+# Use the base image
+FROM hubblo/scaphandre:latest
+
+# Install curl
+RUN apt-get update && apt-get install -y curl
+
+# Optionally clean up apt cache
+RUN rm -rf /var/lib/apt/lists/*
```

Comment:
```suggestion
# Install curl
RUN apt-get install -y curl && \
         apt-get clean
```
Each run creates a seperate layer so removing files in created by the previous run won't yield smaller images. 

Additionally, lets not run `apt-get update` without a good reason since it affects the time taken to build the container.

Reply:
I should mention that it is necessary to update. There is a scenario when building the image for the first time that without an update curl fails to install.